config/resources/users-domain.json

```diff
@@ -34,6 +34,10 @@
         "created": {
           "type": "datetime",
           "predicate": "dct:created"
+        },
+        "is-current-secretary": {
+          "type": "boolean",
+          "predicate": "ext:isHuidigeSecretaris"
```

Comment:
The ticket does not mention it, but I understood that the secretary user has a mandate just like the ministers.
Can we do the same thing as ministers and link a person to a user and then find the correct current mandate of that person?
I might be making this too complex, but I don't see the end goal yet so I want to prevent us from starting on the wrong foot.
This would be the same as setting  "isCurrentPrimeMinister" to true on the user  "Jan Jambon"

Reply:
Not sure, this might have been discusses while I was away. 
As far as I understand the secretary is not a minister. Although they might have a mandate inside the Flemish Government, I'm not sure if we're going to start tracking that inside Kaleidos. I can imagine that changing more often than the ministers, which would add an extra maintenance burden for us. Although it would be very cool complete reflection of the government's personnel as structured data inside Kaleidos/Themis!

> This would be the same as setting "isCurrentPrimeMinister" to true on the user "Jan Jambon"
Yes. Aside from modelling the secretaries as people with a mandate, I was also thinking about using a global "configuration" resource with such data, akin to the `EmailSettings` we currently have. Not sure if it's better, just another option.

Reply:
After today's discussion, that might indeed be the way to go (a new mandate). 

TBD before this is merged